src/sync/sync_manager.hpp

```diff
@@ -85,7 +85,10 @@ friend class SyncSession;
     void set_client_should_reconnect_immediately(bool reconnect_immediately);
     bool client_should_reconnect_immediately() const noexcept;
 
-    /// Force sync client to reconnect immediately if the connection was lost.
+    /// Ask all valid sync sessions to perform whatever tasks might be necessary to
```

Comment:
I'd prefer to keep it this way, at least until we remove the responsibility of the bindings to fetch and refresh tokens. Would changing the name of the method help? You can always call `SyncManager::cancel_reconnect_delay()` if it's the underlying client reconnect delay that you care about.